### PR TITLE
refactor(portal interfaces): renames portal request/response interfac…

### PR DESCRIPTION
**Pull request by tomwayson**
…es to match their functions

AFFECTS PACKAGES:
@esri/arcgis-rest-portal
@esri/arcgis-rest-service-admin

BREAKING CHANGE:
removes "Request" from portal option interfaces & renames response interfaces

### CODE REVIEW

**Comment by tomwayson** [comment]
Let the great 🧀  hunt begin!

File: packages/arcgis-rest-portal/src/items/protect.ts [hunk 1]
```diff
@@ -4,17 +4,15 @@
...
 /**
  * Protect an item. See the [REST Documentation](https://developers.arcgis.com/rest/users-groups-and-items/protect.htm) for more information.
  *
  * @param requestOptions - Options for the request
  * @returns A Promise to protect an item.
  */
-export function protectItem(
-  requestOptions: IItemIdRequestOptions
-): Promise<any> {
+export function protectItem(requestOptions: IUserItemOptions): Promise<any> {
```
Comment (jgravois) [inline packages/arcgis-rest-portal/src/items/protect.ts]: `Promise<{"success": boolean}>`

File: packages/arcgis-rest-portal/src/items/protect.ts [hunk 2]
```diff
@@ -28,9 +26,7 @@ export function protectItem(
  * @param requestOptions - Options for the request
  * @returns A Promise to unprotect an item.
  */
-export function unprotectItem(
-  requestOptions: IItemIdRequestOptions
-): Promise<any> {
+export function unprotectItem(requestOptions: IUserItemOptions): Promise<any> {
```
Comment (jgravois) [inline packages/arcgis-rest-portal/src/items/protect.ts]: `Promise<{"success": boolean}>`

File: packages/arcgis-rest-portal/src/items/remove.ts [hunk 3]
```diff
@@ -26,9 +26,7 @@ import {
  * @param requestOptions - Options for the request
  * @returns A Promise that deletes an item.
  */
-export function removeItem(
-  requestOptions: IItemIdRequestOptions
-): Promise<any> {
+export function removeItem(requestOptions: IUserItemOptions): Promise<any> {
```
Comment (jgravois) [inline packages/arcgis-rest-portal/src/items/remove.ts]: `Promise<{"success": boolean, "itemId": string }>`

File: packages/arcgis-rest-portal/src/users/notification.ts [hunk 4]
```diff
@@ -58,7 +58,7 @@ export function getUserNotifications(
  * @returns A Promise that will resolve with the success/failure status of the request
  */
 export function removeNotification(
-  requestOptions: INotificationIdRequestOptions
+  requestOptions: IRemoveNotificationOptions
 ): Promise<any> {
```
Comment (jgravois) [inline packages/arcgis-rest-portal/src/users/notification.ts]: `Promise<{"success": boolean, "notificationId": string }>`

File: packages/arcgis-rest-portal/src/users/invitation.ts [hunk 5]
```diff
@@ -139,14 +139,14 @@ export function acceptInvitation(
  * @returns A Promise that will resolve with the success/failure status of the request
  */
 export function declineInvitation(
-  requestOptions: IInvitationRequestOptions
+  requestOptions: IGetUserInvitationOptions
 ): Promise<any> {
```
Comment (jgravois) [inline packages/arcgis-rest-portal/src/users/invitation.ts]: `Promise<{
  "success": boolean,
  "username": string,
  "groupId": string,
  "id": string
}>`

File: packages/arcgis-rest-portal/src/users/invitation.ts [hunk 6]
```diff
@@ -111,15 +111,15 @@ export function getUserInvitation(
  * @returns A Promise that will resolve with the success/failure status of the request
  */
 export function acceptInvitation(
-  requestOptions: IInvitationRequestOptions
+  requestOptions: IGetUserInvitationOptions
 ): Promise<any> {
```
Comment (jgravois) [inline packages/arcgis-rest-portal/src/users/invitation.ts]: `Promise<{ "success": boolean, "username": string, "groupId": string, "id": string }>`

**Pull request merged**